docs/valhalla_service.md

```diff
@@ -1,27 +1,27 @@
 
-Welcome to Valhalla open source routing. By now you have perhaps seen our demo page and maybe even checked out our open source software at https://github.com/valhalla. If you are here you are likely interested in accessing the Valhalla routing service at Mapzen to integrate routing and navigation into a Web or mobile application. This page documents the inputs and ouptuts to our service.
+Valhalla is a free, open-source routing service that lets you integrate routing and navigation into a web or mobile application. This page documents the inputs and outputs to the service.
 
-The Mapzen Valhalla routing service is a work in progress. Please check back and be on the lookout for blogs about new features.
+The Valhalla routing service is in active development. You can follow the [Mapzen blog](https://mapzen.com/blog) to get updates.
 
-TODO - Add a note about where to send issues, feature requests, etc.
+To report software issues or suggest enhancements, you can open an issue in GitHub (use the [Thor repository](https://github.com/valhalla/thor) for comments about route paths or [Odin repository](https://github.com/valhalla/odin) for narration). You can also send a message to routing@mapzen.com.
 
 ##### Developer Keys
-Your first step is to get a developer key from Mapzen. This does great things for you! Baldur/Mike please elaborate and provide instructions!
+To use the Valhalla routing service, you must obtain a free developer API key from Mapzen. Sign in at https://mapzen.com/developers to create and manage your API keys.
 
 ##### Notes on Service Limits
 
-Valhalla @ mapzen.com is a free, shared routing service. As such there are rate limits and other limitations that prevent any one user from degrading the overall system performance. We hope to increase rate limits and generally increase the service capabilities through time. In the meantime if the following limits do not work for you please contact us and we will see what we can do to help you. The following limits are in place:
+Valhalla is a free, shared routing service. As such, there are limitations on requests, maximum distances, and numbers of locations to prevent individual users from degrading the overall system performance. Limits may be increased in the future, but the following are currently in place:
 
 * XXX requests per second.
 * YYY requests per day.
-* Pedestrian routes have a limit of ZZZ kilometers.
-* Bicycle routes have a limit of TTT kilometers.
-* Automobile routes have a limit of UUU kilometers.
-* No more than 2 locations can be provided.
+* Pedestrian routes have a limit of 100 kilometers.
+* Bicycle routes have a limit of 500 kilometers.
+* Automobile routes have a limit of 5,000 kilometers.
+* No more than two locations can be provided.
 
-We hope to relax the limits on route distance as we improve performance on longer routes. The limit of number of locations will be increased shortly, stay tuned.
+Send a message if you need higher limits in the meantime.
```

Comment:
Thinking this should be an email, rather than a GitHub issue...will add the address here.